src/main/scala/Matrix.scala

```diff
@@ -1,8 +1,39 @@
 import scala.util.Random
 
-object Matrix {
+object Matrix extends App {
+
+  val alphanum = "abcdefghijklmnopqrstuvwxyz1234567890"
+  var i = 0
+  val randomLine =(70 to 110).toArray
+
+  def randomPick(): Char = {
+    val r = new scala.util.Random
+    val r1 = r.nextInt(alphanum.length)
+    alphanum(r1)
+  }
+
+  def randomChange(): Int = {
+    val r = new scala.util.Random
+    val r1 = r.nextInt(randomLine.length)
+    randomLine(r1)
+  }
+
+    var stop = false
+    var change = randomChange()
+
   def matrix() : Unit = {
-    var alphanum = Random.alphanumeric
-    println(alphanum.nextInt())
+    while(!stop) {
+      print(randomPick())
+      //if (readLine() == "q") stop = true
```

Comment:
why is this commented out? if you're not reassigning the value to "stop" it could be a val